async-http-client-backend/fs2/src/main/scala/sttp/client3/asynchttpclient/fs2/AsyncHttpClientFs2Backend.scala

```diff
@@ -119,46 +119,41 @@ object AsyncHttpClientFs2Backend {
         apply[F](
           AsyncHttpClientBackend.defaultClient(options),
           closeClient = true,
-          blocker,
           customizeRequest,
           webSocketBufferCapacity
         )
       )
 
   /** Makes sure the backend is closed after usage.
     */
-  def resource[F[_]: ConcurrentEffect: ContextShift](
-      blocker: Blocker,
+  def resource[F[_]: Async: Dispatcher](
       options: SttpBackendOptions = SttpBackendOptions.Default,
       customizeRequest: BoundRequestBuilder => BoundRequestBuilder = identity,
       webSocketBufferCapacity: Option[Int] = AsyncHttpClientBackend.DefaultWebSocketBufferCapacity
   ): Resource[F, SttpBackend[F, Fs2Streams[F] with WebSockets]] =
-    Resource.make(apply(blocker, options, customizeRequest, webSocketBufferCapacity))(_.close())
+    Resource.make(apply(options, customizeRequest, webSocketBufferCapacity))(_.close())
 
-  def usingConfig[F[_]: ConcurrentEffect: ContextShift](
-      blocker: Blocker,
+  def usingConfig[F[_]: Async: Dispatcher](
       cfg: AsyncHttpClientConfig,
       customizeRequest: BoundRequestBuilder => BoundRequestBuilder = identity,
       webSocketBufferCapacity: Option[Int] = AsyncHttpClientBackend.DefaultWebSocketBufferCapacity
   ): F[SttpBackend[F, Fs2Streams[F] with WebSockets]] =
     implicitly[Sync[F]].delay(
```

Comment:
you can use `Sync[F].delay`

Reply:
https://github.com/softwaremill/sttp/pull/738/commits/bb1536e139f6762addaf22fc9e8e2058ce16dfdf